src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -18,5 +18,9 @@
         - Additional information about change #1
 -->
 ## Current Release
-
-## Version 2.3.0
\ No newline at end of file
+    * Overview of change #3213
```

Comment:
Please do not describe the PR, but the change in SQL cmdlets, from a customer perspective

- WHat cmdlets changed, and how the api changed (added parameters, parameter set, new proeprties?)
- What the new functionality is

Also, please add this commit to your rpevious PR.